extensions/agent/src/dialogs/jobStepDialog.ts

```diff
@@ -60,13 +60,23 @@ export class JobStepDialog extends AgentDialog<JobStepData> {
 	private readonly AllFilesLabelString: string = localize('jobStepDialog.allFiles', 'All Files (*)');
 
 	// Dropdown options
-	private readonly TSQLScript: string = localize('jobStepDialog.TSQL', 'Transact-SQL script (T-SQL)');
-	private readonly Powershell: string = localize('jobStepDialog.powershell', 'PowerShell');
-	private readonly CmdExec: string = localize('jobStepDialog.CmdExec', 'Operating system (CmdExec)');
-	private readonly AgentServiceAccount: string = localize('jobStepDialog.agentServiceAccount', 'SQL Server Agent Service Account');
-	private readonly NextStep: string = localize('jobStepDialog.nextStep', 'Go to the next step');
-	private readonly QuitJobReportingSuccess: string = localize('jobStepDialog.quitJobSuccess', 'Quit the job reporting success');
-	private readonly QuitJobReportingFailure: string = localize('jobStepDialog.quitJobFailure', 'Quit the job reporting failure');
+	public static readonly TSQLScript: string = localize('jobStepDialog.TSQL', 'Transact-SQL script (T-SQL)');
```

Comment:
Blech, I hate that we don't have a better way of doing this mapping. It's a super common thing too. Could you spend a little time searching around to see if there's some common solution for mapping an enum to a localized string value and vice versa? Not a blocker if nothing obvious comes up but it'd be cool if we came up with a generic solution - in SMO I was able to solve this with C# attributes and a TypeConverter but I don't think Typescript has the same ability to annotate enum values.